storage/storage.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+import os
+import logging
+
+class Storage:
+
+    def __init__(self, blob):
+        self.blob = blob
```

Comment:
Dependency injection of blob, for the sake of clean testing, however its possible we init blob here, and still be able to mock it in tests. Not sure which is best